pkg/sync/v5/data/ConfigMap/openshift-azure-logging/fluentd-config.yaml

```diff
@@ -9,19 +9,58 @@ data:
       </storage>
       tag logs
     </source>
-    <match logs>
+     # drop unused columns
+    <filter logs>
+      @type record_transformer
+      remove_keys _BOOT_ID,_CAP_EFFECTIVE,_MACHINE_ID,_SOURCE_REALTIME_TIMESTAMP,_STREAM_ID,_TRANSPORT,_SOURCE_MONOTONIC_TIMESTAMP
+    </filter>
```

Comment:
we already have 30+ columns. I think we can drop even more.

Reply:
is anyone complaining about these?  I'm inclined just to leave them in for now

Reply:
just me. The geneva UI with all the output just trips me :)

Reply:
please leave them in for now